.github/workflows/ci.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build and publish to ECR
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
```

Comment:
Consider adding `docs/**` (i.e. everything under docs) and `*.md` (i.e. markdown files in the root directory only).

Reply:
I was advised not to add that previously by Kevin, if recall correctly it maybe something that is needed.

Reply:
Ok no probs, not a big deal.